.circleci/config.yml

```diff
@@ -3,7 +3,9 @@ jobs:
   build:
     working_directory: ~/metashare
     docker:
-      - image: circleci/python:3.7.3-node
+      - image: wlonk/oddbird
+    environment:
+      VIRTUAL_ENV: ./venv
```

Comment:
@wlonk What do you think about setting this in CircleCI instead of repeating it here?

Reply:
Unless the value is secret, I much prefer setting it in the config, so it can be in source control and tracked.